BDArmory/Weapons/Missiles/MultiMissileLauncher.cs

```diff
@@ -442,7 +608,8 @@ public void PopulateMissileDummies(bool refresh = false)
                 {
                     if (missileSpawner.ammoCount > i || isClusterMissile)
                     {
-                        if (launchTransforms[i].localScale != Vector3.one) launchTransforms[i].localScale = Vector3.one;
+                        if (launchTransforms[i].localScale != new Vector3((1 / Scale), (1 / Scale), (((1 / Scale) * (1 / Length)) * Scale))); 
```

Comment:
Lots of unnecessary parentheses here.
The third value simplifies to just `1 / Length`.
The `;` shouldn't be there.